config/brokers/mt-channel-broker/deployments/controller.yaml

```diff
@@ -42,6 +42,7 @@ spec:
 
       serviceAccountName: eventing-controller
 
+      enableServiceLinks: false
```

Comment:
nits: while we are here, can you remove the extra newline?

Reply:
Thanks for your comments.  Addressed it.